### PR TITLE
KON-125: Export data til KL

### PR DESCRIPTION
**Pull request by rimi-itk**

### CODE REVIEW

File: Export/KL/Export.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Kun afsluttede sager overføres til KL',
+            'Overføres ikke til KL',
+        ]);
+
+        $this->writeHeader([
+            'Sag',
```
Comment (tuj) [inline Export/KL/Export.php]: Translations in this file?

File: Controller/ExportController.php [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    private function getExportKey(AbstractExport $export)
+    {
+        return md5(\get_class($export));
+    }
+
+    private function buildParameterForm(AbstractExport $export)
+    {
+        $parameters = $export->getParameters();
+
+        $builder = $this->get('form.factory')->createNamedBuilder('parameters_'.md5(\get_class($export)));
```
Comment (tuj) [inline Controller/ExportController.php]: What is this?

Comment (tuj) [reply Controller/ExportController.php]: Use dependency injection for this.

File: Resources/views/export/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    </div>
+
+    </form>
+{% endblock %}
+
+{% block javascripts %}
+  {{ parent() }}
+
+  <script>(function($) {
+     $(document).ready(function() {
```
[the remark this answers is not in the document]
Comment (rimi-itk) [reply Resources/views/export/index.html.twig]: Done.

File: Resources/views/export/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+       var exportIds = {{ exports|keys|json_encode()|raw }},
+           updateUI = function() {
+             // Disable all parameter form controls
+             $('.export-parameters :input').prop('disabled', true);
+             $('.export-parameters').hide();
+             var id = $('#export').val();
```
[the remark this answers is not in the document]
Comment (rimi-itk) [reply Resources/views/export/index.html.twig]: Done.

File: Resources/views/export/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            <option value="{{ key }}">{{ export.title }}</option>
+          {% endfor %}
+        </select>
+      </div>
+    </div>
+
+    {% for name, form in parameterForms %}
+        {% form_theme form with 'bootstrap_3_horizontal_layout.html.twig' %}
+        <div class="export-parameters" id="export-parameters-{{ name }}" style="display: none">
+            {# Show startdata and enddata in a single row called "Period" #}
```
[the remark this answers is not in the document]
Comment (rimi-itk) [reply Resources/views/export/index.html.twig]: Fixed.

File: Controller/ExportController.php (hunk 2 above)
Comment (rimi-itk) [reply Controller/ExportController.php]: Done.

File: Export/KL/Export.php (hunk 1 above)
Comment (rimi-itk) [reply Export/KL/Export.php]: @tuj, experience tells me that it doesn't really add any value to use translations in data exports. There may be very strict requirements for the labels and other stuff in the exported data.

**Pull request merged**